scripts/configs/pg4.yaml

```diff
@@ -0,0 +1,53 @@
+# PG4 experiments.
+---
+APPROACHES:
+  pg3:
+    NAME: "pg3"
+    FLAGS: []
```

Comment:
this was my fault for doing this earlier -- can you actually change all instances of `FLAGS: []` to `FLAGS: {}`? since it's a dict not a list

Reply:
Sounds good!